packages/slate/src/models/range.js

```diff
@@ -657,6 +657,10 @@ class Range extends Record(DEFAULTS) {
    */
 
   moveToRangeOf(start, end = start) {
+    if (this.isBackward) {
+      return this.flip().moveAnchorToStartOf(start)
+        .moveFocusToEndOf(end)
+    }
```

Comment:
Can you change this to only have a single code path, so we're not duplicating the `moveAnchorToStartOf` stuff? For example:

```
let range = this

if (this.isBackward) {
  range = this.flip()
}

return range
  ...
```

Reply:
Fixed. thanks